nucliadb_cluster/src/cluster.rs

```diff
@@ -18,6 +18,13 @@ use uuid::Uuid;
 
 use crate::error::Error;
 
+const NODE_TYPE_KEY: &str = "node_type";
+const LOAD_SCORE_KEY: &str = "load_score";
+
+pub trait Score {
```

Comment:
I dont think this trait is a good abstraction, I would expect for a type implementing `Score` to be already a score, not to be in charge of computing a score. Maybe the function should be called `score` or `value`.

Reply:
Done in 1e220ac8f0c0883c27cb301ea528561520c2106b